flag/service/resources/resources.go

```diff
@@ -0,0 +1,5 @@
+package resources
```

Comment:
The flag structure defines blocks which are always singular.

Reply:
Yeah, fixed.